keyboards/e6_rgb/info.json

```
@@ -0,0 +1,22 @@
+{
+  "keyboard_name": "e6",
+  "url": "",
+  "maintainer": "qmk",
+  "bootloader": "",
+  "width": 15,
+  "height": 5,
+  "layouts": {
+      "LAYOUT_60_wkl": {
+          "key_count":62,
```

You actually only have 61 keys here:

```suggestion
          "key_count":61,
```

---

fixed, thanks